tests/test_transform_inline.py

```diff
@@ -607,3 +609,145 @@ def test_inline_member_routines_with_associate(frontend):
 
     assocs = FindNodes(Associate).visit(routine.body)
     assert len(assocs) == 2
+
+
+@pytest.mark.parametrize('frontend', available_frontends(xfail=[(OMNI, 'No header information in test')]))
```

Comment:
Why should this not work for OMNI? If you parse the module first, the missing xmod is auto-generated and the driver should parse successfully.

Reply:
Correct, wasn't thinking! Many thanks for picking this out!